### PR TITLE
Create, view, edit collections

### LINKED ISSUE #19: Collections detail / edit page
See https://docs.google.com/drawings/d/1r7CJfE7dBZ8Z_yUhL4HG7ehuKI1Is_Kzew0y6VMRnik/edit?usp=sharing

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #19 and #41 

Index Screen:

Screen Shot 2019-08-14 at 11 41 14 AM

New Collection entry:

Screen Shot 2019-08-14 at 11 42 16 AM

Collection Edit/View:

Screen Shot 2019-08-14 at 11 42 49 AM

### CODE REVIEW

File: app/assets/javascripts/collections.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (jcoyne) [inline app/assets/javascripts/collections.coffee]: remove this file.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1,2 +1,3 @@
 // Custom bootstrap variables must be set or imported *before* bootstrap.
-@import "bootstrap";
\ No newline at end of file
+@import "bootstrap";
```
Comment (jcoyne) [inline app/assets/stylesheets/application.scss]: we're not using this file as we're not using sprockets.  Please move it to app/javascripts (for webpacker)

File: app/assets/stylesheets/collections.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the collections controller here.
```
Comment (jcoyne) [inline app/assets/stylesheets/collections.scss]: Remove this file

File: app/helpers/collections_helper.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# A helper for Collections
+module CollectionsHelper
```
Comment (jcoyne) [inline app/helpers/collections_helper.rb]: remove this file.

File: app/assets/javascripts/application.js [hunk 5]
```diff
@@ -16,4 +16,5 @@
 //= require jquery3
 //= require popper
 //= require bootstrap-sprockets
+//= require bootstrap-datepicker
```
Comment (jcoyne) [inline app/assets/javascripts/application.js]: This is not used since we're using webpacker.

File: config/settings/test.yml [hunk 6]
```diff
@@ -21,3 +21,5 @@ wasapi_providers:
         accountid: 1
         username: user
         password: pass
+# Default umber of months to embargo a collection
```
Comment (jcoyne) [inline config/settings/test.yml]: typo.

File: config/settings/test.yml [hunk 7]
```diff
@@ -21,3 +21,5 @@ wasapi_providers:
...
+embargo: 6
```
Comment (jcoyne) [inline config/settings/test.yml]: can this be `default_embargo_months`

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (aaron-collier) [reply app/assets/stylesheets/application.scss]: Yeah, working on that. Missed it before I moved out of draft. Thanks.

File: spec/controllers/collections_controller_spec.rb [hunk 8]
```diff
@@ -5,8 +5,8 @@
 RSpec.describe CollectionsController do
   describe 'GET index' do
     it 'assigns @collections ordered by title' do
-      collection1 = Collection.create title: 'zzz collection'
-      collection2 = Collection.create title: 'aaa collection'
+      collection1 = Collection.create title: 'zzz collection', druid: 'druid:abc123', embargo_months: 6, active: true
```
Comment (jcoyne) [inline spec/controllers/collections_controller_spec.rb]: can you use the factory here?  `create(:collection)`?

File: config/routes.rb [hunk 9]
```diff
@@ -5,5 +5,5 @@
   # root to: 'collections#index'
   get '/', to: redirect('/collections')
 
-  resources :collections, only: %i[index show]
+  resources :collections
```
Comment (jcoyne) [inline config/routes.rb]: We're not implementing `destroy`, so can we enumrate the routes we want?

File: config/settings/development.yml [hunk 10]
```diff
@@ -21,3 +21,5 @@ wasapi_providers:
         accountid: 1
         username: user
         password: pass
+# Default number of months to embargo a collection
+default_embargo_months: 6
```
Comment (jcoyne) [inline config/settings/development.yml]: Can we move this value to `config/settings.yml`?

File: spec/controllers/collections_controller_spec.rb (hunk 8 above)
Comment (aaron-collier) [reply spec/controllers/collections_controller_spec.rb]: Will do.

**Pull request merged**